Multiplicity.Packets/UpdatePlayer.cs

```diff
@@ -1,86 +1,121 @@
-﻿using System;
+using System;
 using System.IO;
+using Multiplicity.Packets.Extensions;
 
 namespace Multiplicity.Packets
 {
-	[Flags]
-	public enum PlayerControlFlags : byte
```

Comment:
The removal of these flags makes it somewhat harder to determine which controls a player is pressing. I'd rather use the built-in flag comparisons in C# rather than the bitflag checking extensions. 

Plus, these flags add some semantics to the bit places. I think they should be returned and utilized.